### ISSUE TITLE
Export & Import to excel

### ISSUE THREAD

**Issue by themsaid** [themsaid/laravel-langman#31]
For easily dealing with translators.

**Comment by vpratfr** [comment]
Rather than Excel, you may want to consider other file formats which are read by tools adapted to translations. For instance the PO file format which can then be edited with POEdit (a free tool).

I don't know if that would work but I think that should not be too hard (PO files are text files). And on the translators side, that would be much more convenient (POEdit has lots of handy features like automatic translation for text he already has in his dictionnary, etc.)

**Comment by mustafaaloko** [comment]
@themsaid I was starting to work on a Laravel package to do this (import/export from/to Excel and Google Sheets). If you want, I can contribute on this in this package instead of creating a new one. Thoughts?

**Comment by themsaid** [comment]
Well, that would be great :)

**Comment by mustafaaloko** [comment]
So you haven't started anything on this? should I scaffold things?

**Comment by mustafaaloko** [comment]
BTW, very and very quick response ;)

**Comment by themsaid** [comment]
Nope, I haven't worked on it yet, but such addition is really helpful and it'll save a lot of time :)

**Comment by vpratfr** [comment]
Something that would really help : https://github.com/oscarotero/Gettext

**Comment by tpraxl** [comment]
+1 I would very much appreciate the proposed enhancement
